### ISSUE TITLE
[docs] nn.SyncBatchNorm missing in online docs

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#18983]
`nn.SyncBatchNorm` docs are missing from master [online docs](https://pytorch.org/docs/master/search.html?q=SyncBatchNorm&check_keywords=yes&area=default#), despite [doc string](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/batchnorm.py#L322-L406) in code.

Indeed, no SyncBatchNorm in [nn.rst](https://github.com/pytorch/pytorch/blob/master/docs/source/nn.rst#batchnorm1d).

A proposal: making a public docs html page "DocTODO" that lists all public classes (without underscore in name) that are missing from rst files and maybe functions as well, along with some color mark if doc string is already present. Like this it's super easy to find these cases and renders pointless creating issues like this one.

**Comment by soumith** [comment]
@vadimkantorov how exactly can we autopopulate DocTODO?

**Comment by vadimkantorov** [comment]
sth like this would work?
```
rst = '\n'.join(open(f).read() for f in ['nn.rst', 'other.rst'])
'doctodo = [member for member in dir(torch) + dir(torch.nn) + dir(torch.nn.functional) if member.__name__ not in rst]'
```
